samples/bluetooth/peripheral_hr/prj.conf

```diff
@@ -8,3 +8,5 @@ CONFIG_BT_BAS=y
 CONFIG_BT_HRS=y
 CONFIG_BT_DEVICE_NAME="Zephyr Heartrate Sensor"
 CONFIG_BT_DEVICE_APPEARANCE=833
+
+CONFIG_GPIO=y
```

Comment:
Does enabling this for a board that does not have LED, increase memory usage without actually adding any new functionality?

Reply:
Not able to test, as all my boards have LED. But agree that logically there could be more code added to boards without LED when GPIO module being included.

Removed explicit enable in the prj.conf, but depend on boards that already enable GPIO be default, like most nRF DK boards. In this case, LED  will be used from devicetree, when enabled by default.